pilot/pkg/networking/core/v1alpha3/route/route_test.go

```diff
@@ -537,7 +537,7 @@ func TestBuildHTTPRoutes(t *testing.T) {
 		})
 
 		proxy := node(cg)
-		hashByDestination := route.GetConsistentHashForVirtualService(cg.PushContext(), proxy, virtualServicePlain)
+		hashByDestination := route.GetConsistentHashForVirtualService(cg.PushContext(), proxy, virtualServicePlain, map[host.Name]*model.Service{})
```

Comment:
How about provide a `nil` map instead of a `empty` map and add an assertion in `GetConsistentHashForVirtualService`?